app/controllers/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  
+  def show
+    @user = User.find(params[:id])
```

Comment:
since they are signed in, let's use current_user here, instead of finding the user by id. This is also better for security. If you find the user by id, then I can go to your profile. This page is where we will eventually list todo list items, so that means I would be able to see/delete your to do list items.